libdevice/crt_wrapper.cpp

```diff
@@ -9,6 +9,14 @@
 #include "wrapper.h"
 
 #ifdef __SPIR__
+DEVICE_EXTERN_C
```

Comment:
I guess we'd better have a dedicated integer math file instead of placing this in the `crt` portion, but we can do the clean-up later.

Reply:
> I guess we'd better have a dedicated integer math file instead of placing this in the `crt` portion, but we can do the clean-up later.

It seems that "div" is declared in <cstdlib> instead of <cmath> but it makes sense to group all "math related" items into one math file. I will update it later.
Thanks.